release.yaml

```diff
@@ -29,9 +29,11 @@ rancher-alerting-drivers:
 rancher-cis-benchmark:
   - 105.2.0+up7.2.0-rc.2
   - 105.2.0+up7.2.0-rc.1
+  - 105.2.0+up7.2.0
 rancher-cis-benchmark-crd:
   - 105.2.0+up7.2.0-rc.2
   - 105.2.0+up7.2.0-rc.1
```

Comment:
@nicholasSUSE It is not removing the older rc entries from the release.yaml.. These need to be removed